packages/structures/src/channels/mixins/ThreadChannelMixin.ts

```diff
@@ -0,0 +1,45 @@
+import type { ThreadChannelType } from 'discord-api-types/v10';
+import { kData } from '../../utils/symbols.js';
+import type { Channel, ChannelDataType } from '../Channel.js';
+
+export interface ThreadChannelMixin<
+	Type extends ThreadChannelType = ThreadChannelType,
+	Omitted extends keyof ChannelDataType<Type> | '' = '',
```

Comment:
This is the only mixin that has omitted, I can't remember if it actually is necessary to have it in the mixins or not.